CHANGELOG.md

```diff
@@ -35,7 +106,7 @@ All notable changes to this project will be documented in this file. For future
 - Added a warning for empty IQN matrix, but keep the simulation going. This allows to start from a zero initial state that only later changes (e.g., an opening valve in a flow simulation) (https://github.com/precice/precice/pull/1895)
 - Added an optional preconditioner to the Aitken underrelaxation acceleration. (https://github.com/precice/precice/pull/1379)
 - Added check for duplicated `<exchange/>` tags. (https://github.com/precice/precice/pull/1467)
-- Added communication of substeps data for waveform coupling. A new attribute `substeps` in the `exchange` tag allows activating and deactivating this additional communication. `substeps` defaults to `false`, since not yet all acceleration schemes support this feature. This default may change once preCICE is fully ported to handling waveforms.  See https://github.com/precice/precice/issues/1825 for details. (https://github.com/precice/precice/pull/1664)
```

Comment:
Why this content change in the changelog of a past release?

Reply:
https://github.com/precice/precice/pull/2244#pullrequestreview-2712715696